README.md

```diff
@@ -338,10 +338,10 @@ If you fork the project on GitHub, you can run your fork's [build workflow](.git
     --plugin-dirs PATH              Path to an additional directory to search
                                     for plugins. This option can be used
                                     multiple times to add multiple directories.
-                                    Note that this currently only works for
-                                    extractor plugins; postprocessor plugins can
-                                    only be loaded from the default plugin
-                                    directories
+                                    Add "no-external" to disable searching
+                                    default external plugin directories (outside
+                                    of python environment)
```

Comment:
How is this useful as opposed to `--no-plugins`?

Reply:
Up for debate. This will allow PYTHON_PATH to still be searched, but no other place to be searched.